// [[Reason: TSC adds type reference directives.]] ////

//// [tests/cases/conformance/node/allowJs/nodeModulesAllowJsImportHelpersCollisions2.ts] ////

===================================================================
--- TSC	declarations
+++ DTE	declarations
@@ -1,8 +1,7 @@
 
 
 //// [/.src/out/index.d.ts]
-/// <reference path="../types.d.ts" />
 export * from "fs";
 export * as fs from "fs";
 //# sourceMappingURL=index.d.ts.map
 /// [Errors] ////
